stories/sign-in-button.stories.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
```

Comment:
I think this file is the same as stories/signin-signup.stories.js  and this one should be removed.